### ISSUE TITLE
AI for Windows is unable to detect text color for short text strings

### ISSUE THREAD

**Issue by roryfa** [microsoft/accessibility-insights-windows#996]
Please check whether the bug has [already been filed](https://github.com/Microsoft/accessibility-insights-windows/issues).

**Describe the bug**
I noticed that the auto-detect color contrast feature in the latest Accessibility Insights win32 app is unable to detect colors in our app UI or content in the web container.

![image](https://user-images.githubusercontent.com/54419365/100637967-4f598380-32e8-11eb-93c1-1e47d3ddf13f.png)

**To Reproduce**
1. Open Microsoft Edge (Anaheim)
2. Open AI for Windows
3. Attempt to detect color contrast ratio for a button on the application toolbar

**Expected behavior**
AI for windows accurate determines text color and background fill color

**Actual behavior**
AI for windows says it is unable to detect colors

**Comment by ferBonnin** [comment]
thanks for the bug @roryfa! From looking at this use case, testing icons is not an available functionality for the auto-detect feature. From our [documentation on Color Contrast](https://accessibilityinsights.io/docs/en/windows/getstarted/colorcontrast/#manually-check-contrast-ratio): "_Because auto detect uses a heuristic intended specifically for text, the contrast ratio of UI components and graphical elements must be tested manually_"

**Comment by DaveTryon** [comment]
@roryfa, I suspect that this is the same root cause as #537. Please take a look and close this out if you agree. If you disagree, please update your repro steps with a text field (as opposed to an icon). Thanks!

**Comment by DaveTryon** [comment]
I have a branch where I've rewritten the color contrast algorithm. We'll see if we can get this ready to ship

**Comment by DaveTryon** [comment]
This isn't really in an active state, so we're restarting the triage process on this

**Comment by salux** [comment]
+1 for this getting into the production version!

I just filed a [question](https://github.com/microsoft/accessibility-insights-windows/issues/1403) on this and learned of this bug today. We're running into issues where the new algo might be helpful.

Currently, the tool is providing strange color selections that are actually colors from the antialiasing applied by ClearType. So, even though our code may say the text is black over white, the tool automatically detects a purple color over white. We're having a rough time making things accessible since the tool is giving odd results.

**Comment by DaveTryon** [comment]
@salux, you can also check color contrast by using the eye droppers, as documented at https://www.accessibilityinsights.io/docs/windows/getstarted/colorcontrast/#manually-check-contrast-ratio. It's slower but should allow you to get accurate results until we have a better automated option in place.

**Comment by ferBonnin** [comment]
From the triage discussion with the team, something to consider is that the new algorithm might impact performance. An option would be to create a toggle in Settings that allows users to try the new algorithm, however this might cause discoverability questions.  Leaving this as ready for triage for more conversations next week

**Comment by codeofdusk** [comment]
We've disabled automatic colour contrast analysis by default for now in latest `main`/next canary (see #1441) but keeping this open for tracking.

**Comment by DaveTryon** [comment]
We can't really fix this correctly at this time
